src/main/resources/elasticsearch/index/instance.json

```diff
@@ -33,6 +33,11 @@
         "filter": [ "folio_word_delimiter_graph", "icu_folding" ],
         "char_filter": [ "and_char_replacement" ],
         "type": "custom"
+      },
+      "whitespace_analyzer": {
```

Comment:
whitespace_lowercase probably will make it consistent with other

Reply:
Are you asking if it is consistent with  keyword_lowercase? I guess, that the distinction is that "whitespace" enables full-text searches, whereas "keyword" makes exact matches. Originally, "whitespace" was used for identifiers.value. Therefore, I included an additional analyzer to preserve the original functionality

Reply:
I mean probably will be better to rename it to whitespace_lowercase_analyzer